testdata/generated/storage-indexer/categories.json

```diff
@@ -81,7 +84,8 @@
   },
   {
     "id": "web",
-    "title": "Web",
-    "count": 14
+    "title": "Web Server",
+    "count": 14,
+    "subcategory_of": "infrastructure"
   }
```

Comment:
These will be the changes in the API:
* Titles updated for some categories.
* `subcategory_of` added in subcategories.